dw-list-item.js

```diff
@@ -443,6 +445,39 @@ export class DwListItem extends LitElement {
     }
   }
 
+  _getNextFocusableElement(el) {
+    if(el.nextElementSibling && (el.nextElementSibling.hasAttribute('opened') || el.nextElementSibling.hasAttribute('hasChild'))) {
+      return el.nextElementSibling.children[0];
+    }
+
+    if(el.nextElementSibling) {
+      return el.nextElementSibling 
+    }
+
+    if(el.parentElement && (el.parentElement.hasAttribute('opened') || el.parentElement.hasAttribute('hasChild'))) {
+      return el.parentElement.nextElementSibling;
+    }
+    
+    return el;
+  }
+
+  _getPreviousFocusableElement(el) {
+    if(el.previousElementSibling && (el.previousElementSibling.hasAttribute('opened') || el.previousElementSibling.hasAttribute('hasChild'))) {
```

Comment:
as per standard naming conventions, we don't use camel-case in attribute names. It should be dash separated, all lower cases.